modules/social_features/social_album/social_album.install

```diff
@@ -307,3 +324,51 @@ function social_album_update_11401(): string {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Grant group albums permissions for anonymous users.
+ */
+function social_album_update_11402(): void {
+  /** @var \Drupal\group\Entity\GroupRoleInterface $group_role_anonymous */
+  $group_role_anonymous = \Drupal::entityTypeManager()->getStorage('group_role')->load('flexible_group-anonymous');
+  $group_role_anonymous->grantPermissions(['view group_node:album content', 'view group_node:album entity'])->save();
+}
+
+/**
+ * Grant group albums permissions for group outsiders.
+ */
+function social_album_update_11403(): void {
+  /** @var \Drupal\group\Entity\GroupRoleInterface $group_role_outsider */
```

Comment:
I think we can remove it, if we call `_social_album_set_group_permissions` and changes are added to that function.